new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful and popular editor that has many commands. Unlike most
+text editors where you use the keyboard to enter text and arrow keys to move you
+cursor, Vim uses the keyboard to enter commands to manipulate the text.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these commands
+are not arbitrarily chosen, they form a language with its own grammar.
+
+This tutor is designed to describe enough of the commands
+that you will be able to easily use Vim as an all-purpose editor.  It is
+IMPORTANT to remember that this tutor is set up to teach by use.  That means
```

Comment:
I think a better scheme than highlighting selected words would just be to surround them with asterisks, like markdown. That way we never need to expand our core list of "bold words" because external projects request so.

If this is already done... oops.